src/babashka/json.clj

```diff
@@ -16,14 +16,17 @@
                   cheshire/cheshire
                   'babashka.json.internal.cheshire/fns
                   com.cnuernber/charred
-                  'babashka.json.internal.charred/fns)))
+                  'babashka.json.internal.charred/fns
+                  metosin/jsonista
+                  'babashka.json.internal.jsonista/fns)))
         (or
          ;; this should always work in babashka
          (try-require 'babashka.json.internal.cheshire/fns)
          ;; this needs to be added to the classpath explicitly
          (try-require 'babashka.json.internal.charred/fns)
          ;; this one should always work as this project has a dependency on it.
-         (try-require 'babashka.json.internal.data-json/fns)))]
+         (try-require 'babashka.json.internal.data-json/fns)
+         (try-require 'babashka.json.internal.jsonista/fns)))]
```

Comment:
The jsonista try-require should go before data.json, as data.json always succeeds